css/style.css

```diff
@@ -47,32 +34,139 @@ Developer @GA in NYC
 
 body {
      color: #000;
-     font-size: 12px;
-     line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+     font-family: "open sans";
```

Comment:
Generally a good idea to have fallback fonts, especially when using a google font. The google font may not be able to load for some reason and you want to be prepared for that:

```
font-family: "open sans",helvetica,sans-serif;
```